rln/src/public.rs

```diff
@@ -820,7 +820,7 @@ impl RLN<'_> {
     //
     // output_data is [ proof<128> | root<32> | external_nullifier<32> | x<32> | y<32> | nullifier<32>]
     // we skip it from documentation for now
```

Comment:
This comment is no longer valid

Reply:
thanks, addressed: c4ee1ff14c3b57879bccfc721d5bc4cfbbacc15b